docs/DeveloperGuide.md

```diff
@@ -314,11 +340,102 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The user specifies an invalid index.
```

Comment:
From our current UG, the format for the delete feature is as follows:
Format: `delete [identification] [field]`

I think we are using `[indentification]`, which can be `name` or `id`, instead of `index`.
`[field]` is to be left blank to delete the patient, otherwise the field corresponding to the `[identification]` is deleted.

Reply:
My bad, I had used the given format of the `delete` command and neglected to double-check it with our updated UG.

On this note, the team has discussed this in an offline meeting that we will be using `delete [identification] [field]` to extend on the existing `delete` command. We are likely dropping the use of `delete [index]` as a result.